Test/DafnyTests/TestAttribute.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %dafny /compileVerbose:1 /compile:0 /spillTargetCode:3 /noVerify "%s" > "%t"
+// RUN: %dafny /compileVerbose:1 /compile:0 /spillTargetCode:3 /noVerify /synthesize "%s" > "%t"
 // RUN: dotnet test -v:q -noLogo %S 2> %t.testresults.raw || true
 // Remove the timestamp prefixes on the expected errors
 // RUN: sed 's/[^]]*\]//' "%t".testresults.raw >> "%t"
```

Comment:
Off-topic, but I just realized this post-processing can be replaced with OutputCheck-style `// CHECK:` directives now that I've added that support to the xUnit lit test runner :)

Reply:
I am now using the `//CHECK:` directive